about.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>PILAR TORRES</title>
+  <link rel="stylesheet" href="styles/main.css">
+  <link rel="stylesheet" href="index.html">
+  <link rel="stylesheet" href="portfolio.html">
+  <link rel="stylesheet" href="contact.html">
+</head>
+
+<body>
+  <header class="content-header" style="padding:128px 16px">
```

Comment:
`style="padding:128px 16px"` please try not to use the style attribute directly in your html files. If you want to debug something you can use the `element.style` rule in chrome dev tools.